src/http/client/response.cr

```diff
@@ -31,6 +31,10 @@ class HTTP::Client::Response
     @body
   end
 
+  def successful?
```

Comment:
@luislavena I guess it would be even better! The problem is the consistence between the other helpers: `error?`, `failed?`, `fail?` , `redirect?` what do you think?